schema/config_schema_v3.9.json

```diff
@@ -128,6 +157,15 @@
           }
         },
         "container_name": {"type": "string"},
+        "cpu_count": {"type": "integer", "minimum": 0},
+        "cpu_percent": {"type": "integer", "minimum": 0, "maximum": 100},
+        "cpu_shares": {"type": ["number", "string"]},
+        "cpu_quota": {"type": ["number", "string"]},
+        "cpu_period": {"type": ["number", "string"]},
+        "cpu_rt_period": {"type": ["number", "string"]},
+        "cpu_rt_runtime": {"type": ["number", "string"]},
+        "cpus": {"type": "number", "minimum": 0},
+        "cpuset": {"type": "string"},
```

Comment:
There's an overlap between some of these and `resource.cpus`.

Some of them are really hard to use, and must be specified "together" (`cpu-period` and `cpu-quota`, but also `cpuset-cpus` and `cpuset-mem`). Ideally we would have a better design for that to make such things logically grouped, so that they will be set together https://docs.docker.com/engine/reference/run/#cpu-period-constraint